### PR TITLE
feat(data-attributes): save verified status

### PR DESCRIPTION
**Pull request by karlprieb**

### CODE REVIEW

File: src/database/standalone-sqlite.ts [hunk 1]
```diff
@@ -2329,13 +2337,25 @@ export class StandaloneSqliteDatabaseWorker {
...
-      id: fromB64Url(id),
+      id: dataId,
       parent_id: fromB64Url(parentId),
       data_offset: dataOffset,
       data_size: dataSize,
-      indexed_at: currentUnixTimestamp(),
+      indexed_at: currentTimestamp,
     });
+
+    const parentAttributes = this.getDataAttributes(parentId);
```
Comment (karlprieb) [inline src/database/standalone-sqlite.ts:2351]: @djwhitt Is this the best way to check if parent is verified? If so I think we should create a cache for it as this query is expensive to run for each data item of the same parent

File: src/database/standalone-sqlite.ts [hunk 2]
```diff
@@ -1140,26 +1140,34 @@ export class StandaloneSqliteDatabaseWorker {
...
     });
+
     if (dataRoot !== undefined) {
       this.stmts.data.insertDataRoot.run({
         data_root: fromB64Url(dataRoot),
         contiguous_data_hash: hashBuffer,
-        indexed_at: currentUnixTimestamp(),
+        indexed_at: currentTimestamp,
+        verified: verified ? 1 : 0,
+        verified_at: verified ? currentTimestamp : 0,
```
Comment (karlprieb) [inline src/database/standalone-sqlite.ts:1170]: null instead of 0

File: src/database/standalone-sqlite.ts (hunk 1 above)
Comment (karlprieb) [reply src/database/standalone-sqlite.ts:2351]: check on update query

File: src/database/standalone-sqlite.ts [hunk 3]
```diff
@@ -2329,12 +2339,22 @@ export class StandaloneSqliteDatabaseWorker {
...
-      parent_id: fromB64Url(parentId),
+      id: idBuffer,
+      parent_id: parentIdBuffer,
       data_offset: dataOffset,
       data_size: dataSize,
-      indexed_at: currentUnixTimestamp(),
+      indexed_at: currentTimestamp,
+    });
+
+    this.stmts.data.updateVerifiedDataId.run({
```
Comment (karlprieb) [inline src/database/standalone-sqlite.ts:2354]: @djwhitt do you think it make sense to run this update in saveNestedDataId or just get the `verified` value from parent while running `insertDataId`?

Comment (djwhitt) [reply src/database/standalone-sqlite.ts:2354]: Hmm... good question. I think it's probably better to do it in `insertDataId` and save a query.

Comment (karlprieb) [reply src/database/standalone-sqlite.ts]: [`6ccf079` (#222)](https://github.com/ar-io/ar-io-node/pull/222/commits/6ccf079d351126718a8023c7939b17a68974207d)

**Review by djwhitt — APPROVED** [review]

**Pull request merged**